### PR TITLE
Implement `to_mantissa_scale`

### LINKED ISSUE #4191: Implement `to_mantissa_scale` in `Amount`
To bind Massa Amount to gRPC `NativeAmount` we need to implement a function to make the conversion:

Add the following constants  to `constants.rs`
```rust
pub const AMOUNT_DECIMAL_SCALE: u32 = 9;
pub const AMOUNT_DECIMAL_FACTOR: u64 = 10u64.pow(AMOUNT_DECIMAL_SCALE);
```

Add to `amount.rs`:
```rust
    /// Returns the value in the (mantissa, scale) format where
    /// amount = mantissa * 10^(-scale)
    pub fn to_mantissa_scale(&self) -> (u64, u32) {
        (self.0, AMOUNT_DECIMAL_SCALE)
    }
```

Add some tests.

### PR DESCRIPTION
**Pull request by aoudiamoncef**
* [x] document all added functions
* [ ] try in sandbox /simulation/labnet
* [x] unit tests on the added/changed features
  * [x] make tests compile
  * [x] make tests pass 
* [x] add logs allowing easy debugging in case the changes caused problems
* [x] if the API has changed, update the API specification

closes #4191

### CODE REVIEW

File: massa-models/src/config/constants.rs [hunk 1]
```diff
@@ -101,6 +101,10 @@ pub fn get_period_from_args() -> u64 {
     last_start_period
 }
 
+/// Decimals scale for the amount
+pub const AMOUNT_DECIMAL_SCALE: u32 = 9;
+/// Decimals factor for the amount
+pub const AMOUNT_DECIMAL_FACTOR: u64 = 10u64.pow(AMOUNT_DECIMAL_SCALE);
```
Comment (damip) [inline massa-models/src/config/constants.rs:107]: those constants don't belong here, see line 16 of amount.rs

**Comment by damip** [comment]
@aoudiamoncef note that it was already done by JF here https://github.com/massalabs/massa/pull/4199/files#diff-f088023ab5945158cb5faba2b0c35229bc58b7dd2175ddb3d279706e89a22e7b

**Comment by bilboquet** [comment]
> @aoudiamoncef note that it was already done by @bilboquet here [#4199 (files)](https://github.com/massalabs/massa/pull/4199/files#diff-f088023ab5945158cb5faba2b0c35229bc58b7dd2175ddb3d279706e89a22e7b)

I've copied what Moncef explained in the related issue.
If this small PR is merged quickly, I can rebase on it :)

File: massa-models/src/config/constants.rs (hunk 1 above)
Comment (aoudiamoncef) [reply massa-models/src/config/constants.rs]: done

**Review by damip — APPROVED** [review]
Good !

**Pull request merged**